src/System.IO.Pipelines/tests/PipeWriterTests.cs

```diff
@@ -209,6 +209,8 @@ public async Task WriteAsyncWithACompletedReaderNoops()
                 await pipe.Writer.WriteAsync(writeBuffer);
             }
 
+            Assert.Equal(1, pool.CurrentlyRentedBlocks);
```

Comment:
So this was always failing before, right?

Reply:
Yeah 🙈